lib/g10_certification_test_kit/bulk_data_group_export.rb

```diff
@@ -166,6 +166,7 @@ class BulkDataGroupExport < Inferno::TestGroup
           assert response_body.key?(key), "Complete Status response did not contain \"#{key}\" as required"
         end
 
+        output requires_access_token: response_body['requiresAccessToken'].to_s.downcase
```

Comment:
Do we really have to turn this boolean into a string?  Since the output is stored as JSON it seems like this should be possible

https://github.com/inferno-framework/inferno-core/blob/6614f231150529b79d79003b2753018e9e7d7a24/lib/inferno/test_runner.rb#L96

@Jammjammjamm?

Reply:
I was trying to say you don't need any of `.to_s.downcase` since its a boolean in the JSON response, not a string, so letter case shouldn't matter.

```suggestion
        output requires_access_token: response_body['requiresAccessToken']
```

Reply:
Also, all inputs/outputs are stored as strings in the database.

Reply:
Gotcha. I was misunderstanding the input/output saving and loading